app/src/main/java/com/framgia/forder/screen/main/MainViewModel.java

```diff
@@ -78,6 +79,14 @@ public void onFooterClicked(View view) {
         switch (view.getId()) {
             case R.id.layout_home:
                 setCurrentTab(Tab.TAB_HOME);
+                Navigator navigator = new Navigator(mViewPagerAdapter.getFragment(Tab.TAB_HOME));
+                navigator.goBackFragmentByTag(TAG, POP_BACK_STACK_CLEAR_TASK);
+                MainPageFragment fragment =
+                        (MainPageFragment) mViewPagerAdapter.getFragment(Tab.TAB_HOME)
+                                .getChildFragmentManager()
+                                .getFragments()
```

Comment:
ERROR: [RestrictedApi] FragmentManager.getFragments can only be called from within the same library group (groupId=com.android.support)
 This API has been flagged with a restriction that has not been met.

Examples of API restrictions:
* Method can only be invoked by a subclass
* Method can only be accessed from within the same library (defined by  the Gradle library group id)
.* Method can only be accessed from tests.
.
You can add your own API restrictions with the `@RestrictTo` annotation.